manifest.json

```
@@ -232,33 +237,11 @@
             "type": "boolean",
             "default": false
           },
-          "customHeader": {
-            "title": "admin/store.advancedSettings.customHeader.title",
-            "type": "array",
-            "items": {
-              "type": "object",
-              "properties": {
-                "__editorItemTitle": {
-                  "type": "string",
-                  "title": "admin/store.advancedSettings.customHeader.__editorItemTitle.title",
-                  "description": "admin/store.advancedSettings.customHeader.__editorItemTitle.description"
-                },
-                "key": {
-                  "type": "string",
-                  "title": "admin/store.advancedSettings.customHeader.key.title",
-                  "description": "admin/store.advancedSettings.customHeader.key.description"
-                },
-                "value": {
-                  "type": "string",
-                  "title": "admin/store.advancedSettings.customHeader.value.title",
-                  "description": "admin/store.advancedSettings.customHeader.value.description"
-                }
-              },
-              "required": [
-                "key",
-                "value"
-              ]
-            }
+          "useDefaultBrowserNavigation": {
```

```suggestion
          "useDefaultBrowserNavigation": {
```

Same here, the version is without customHeader